config/cdot_ppmBsm.properties

```diff
@@ -22,10 +22,14 @@ privacy.filter.geofence.ne.lon=-102.051701
 privacy.topic.consumer=topic.OdeBsmJson
 privacy.topic.producer=topic.FilteredOdeBsmJson
 
-group.id=0
+group.id=CDOT_PPM_BSM
```

Comment:
Let's just call the group the same as in ppmBsm.properties, "PPM_BSM". We'll be swapping this over to a more generalized solution to merge into the USDOT repo and this will help with that.

Reply:
The group.id property in this file has been changed to PPM_BSM and the group.id property in cdot_ppmTim.properties has been changed to PPM_TIM.